deployment/ccip/changeset/cs_update_rmn_config.go

```diff
@@ -0,0 +1,256 @@
+package changeset
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/gethwrappers"
+	mcmsWrappers "github.com/smartcontractkit/ccip-owner-contracts/pkg/gethwrappers"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/mcms"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/timelock"
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/common/proposalutils"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/rmn_home"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/rmn_remote"
+)
+
+type SetRMNHomeCandidateConfig struct {
+	HomeChainSelector uint64
+	RMNStaticConfig   rmn_home.RMNHomeStaticConfig
+	RMNDynamicConfig  rmn_home.RMNHomeDynamicConfig
+	DigestToOverride  [32]byte
+}
+
+func (c SetRMNHomeCandidateConfig) Validate() error {
+	if c.HomeChainSelector == 0 {
+		return fmt.Errorf("HomeChainSelector must be set")
```

Comment:
let's add a check for [IsValidChainSelector](https://github.com/smartcontractkit/chainlink/blob/develop/deployment/helpers.go#L157)